packages/camlimages/camlimages.4.2.4/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "1.2"
+version: "4.2.4"
+authors: [
+  "Jun Furuse"
+  "Fran\195\167ois Pessaux"
+  "Pierre Weis"
+]
+maintainer: "jun.furuse@gmail.com"
+homepage: "https://bitbucket.org/camlspotter/camlimages"
+bug-reports: "https://bitbucket.org/camlspotter/camlimages/issues?status=new&status=open"
+dev-repo: "hg://https://bitbucket.org/camlspotter/camlimages"
+build: [
+  [ "omake" "--configure" ]
+]
+install: [
+  [ "omake" "install" ]
+]
+remove: [
+  [ "ocamlfind" "remove" "camlimages" ]
+]
+depends: [
+  "ocamlfind" { build } "base-unix" "omake" { build }
+]
+depopts: [
+  "lablgtk"
+]
+available: [
+  ocaml-version >= "4.02.0"
```

Comment:
This needs to be `>= "4.03.0"` due to the use of

```
# File "images.ml", line 105, characters 10-32:
# Error: Unbound value String.lowercase_ascii
# *** omake: targets were not rebuilt because of errors:
```